### ISSUE TITLE
UKDALE- what happened to meter 1?

### ISSUE THREAD

**Issue by daniprec** [nilmtk/nilmtk#876]
I'm trying to emulate the results of this repository: https://github.com/lmssdd/TPNILM

The authors of it don't use nilmtk to preprocess the data, and use the meter 1 of the UKDALE dataset (buildings 1, 2 and 5), which contains the aggregate load of the household. This meter 1 is present in the raw data of UKDALE.

I do use nilmtk, and found out that meter 1 is missing from the MeterGroups of those houses. Instead, those MeterGroups contains a new meter, called "site_meter" which is used as the main meter. The problem is: the aggregate load from meter 1 and the load from the main meter don't match.

Is there an easy and memory-friendly way of retrieving the aggregate load? I could load all the meters and sum them, but that is very heavy - specially for house 1, which has 53 meters. The original UKDALE data does contain the aggregate load, so I assume nilmtk stores the series somewhere, and I just can't find it.

Thanks in advanced. Good job doing this project!

TL;DR - Raw data of UKDALE contains the "meter 1" with the aggregate load. Nilmtk seems to replace it with the "main meter", which doesn't match the aggregate load.

**Comment by PMeira** [comment]
@daniprec Did you manage to find anything else about this?

The converter for UKDALE doesn't seem to do anything special. The dataset contains both 1- and 6-second sampling rates though, and its metadata has its own repo at https://github.com/JackKelly/UK-DALE_metadata
